Volume/Views/MainView/HomeList.swift

```diff
@@ -75,6 +75,15 @@ struct HomeList: View {
             fetchWeeklyDebrief()
         }
     }
+
+    private var someFollowedArticles: Bool {
```

Comment:
rename to `isFollowingPublications` or something more consistent w/ other booleans in the project.

Reply:
Ok